actionpack/test/dispatch/server_timing_test.rb

```diff
@@ -88,8 +97,8 @@ def create
     headers1 = t1.value[1]
     headers2 = t2.value[1]
 
-    assert_match(/custom.event;dur=\w+/, headers2["Server-Timing"])
-    assert_no_match(/custom.event;dur=\w+/, headers1["Server-Timing"])
+    assert_match(/custom.event;dur=\w+/, headers2["server-timing"])
+    assert_no_match(/custom.event;dur=\w+/, headers1["server-timing"])
```

Comment:
```suggestion
    assert_match(/custom.event;dur=\w+/, headers2[ActionDispatch::Constants::SERVER_TIMING])
    assert_no_match(/custom.event;dur=\w+/, headers1[ActionDispatch::Constants::SERVER_TIMING])
```

These need to be dynamically cased because headers is just a Hash here instead of `Rack::Headers`.

Reply:
🤦